src/components/organisms/footer/_footer.scss

```diff
@@ -83,7 +83,7 @@ body {
 
   transition-property: width, height, margin-top;
   transition-duration: .25s;
-  @extend %easeOutOver;
+  @extend %easeOutQuad;
```

Comment:
`easeOutOver` doesn't exist, so I just switched it to `easeOutQuad`. There may be a better one for it. Know of one T?